skopt/gbt.py

```diff
@@ -50,4 +50,4 @@ def fit(self, X, y):
 
     def predict(self, X):
         """Predictions for each quantile."""
-        return np.vstack([rgr.predict(X) for rgr in self.regressors_])
+        return np.asarray([rgr.predict(X) for rgr in self.regressors_])
```

Comment:
Switched to this as I kept having to write: `a, b, c = e.predict().T`, is there a better way to unpack things?